git.lua

```diff
@@ -112,6 +112,30 @@ local function branches(token)
     return res
 end
 
+local function alias(token)
+    local res = {}
+
+    -- Try to resolve .git directory location
+    local git_dir = get_git_dir()
+
+    if git_dir == nil then return res end
+
+    f = assert (io.popen ("git config --get-regexp alias"))
+
+    for line in f:lines() do
+        local s, e = line:find(" ", 1, true)
```

Comment:
Cool! This can help to add autocompletion on aliases, by detecting if alias starts with a know git command.

    local alias, command = line:match("alias%.(.+)%s+(.+)$")
    --pseudocode
    if start of command is in git_parser autocompletion
        add alias to git_parser with autocompletion matched
    endif
    -pseudocode

That way feature will be complete.